app/controllers/allocation_controller.rb

```diff
@@ -0,0 +1,17 @@
+class AllocationController < ApplicationController
+  before_action :authorise
+
+  def create
+    AllocationService.create_allocation(allocation_params)
+    render_ok
+  end
+
+private
+
+  def allocation_params
+    params.require(:allocation).permit(
+      :staff_id, :offender_no, :offender_id, :allocated_at_tier,
+      :created_by, :prison, :reason, :note, :email
```

Comment:
I'm not sure it's obvious what `reason` means - would `override_reason` be clearer?

Reply:
Change made now @TobyRet